packages/agoric-cli/src/commands/inter.js

```diff
@@ -235,34 +240,105 @@ export const makeInterCommand = (
     }
   };
 
-  const liquidationCmd = interCmd
-    .command('liquidation')
-    .description('liquidation commands');
-  liquidationCmd
+  const auctionCmd = interCmd
+    .command('auction')
+    .description('auction commands');
+  auctionCmd
     .command('status')
     .description(
-      `show amount liquidating, vault manager price
+      `show auction status in JSON format
 
 For example:
 
+inter auction status
 {
-  "liquidatingCollateral": "10 IbcATOM",
-  "liquidatingDebt": "120 IST",
-  "price": "12.00 IST/IbcATOM"
+  "schedule": {
+    "activeStartTime": null,
+    "nextStartTime": "2023-04-19T01:20:02.000Z",
+    "nextDescendingStepTime": "2023-04-19T01:20:02.000Z"
+  },
+  "book0": {
+    "startPrice": "9.99 IST/IbcATOM",
+    "startProceedsGoal": null,
+    "proceedsRaised": null,
+    "startCollateral": "0 IbcATOM",
+    "collateralAvailable": "0 IbcATOM",
+    "currentPriceLevel": "4.995 IST/IbcATOM"
+  },
+  "params": {
+    "DiscountStep": "5.00%",
+    "ClockStep": "00:00:10",
+    "LowestRate": "45.00%"
+  }
 }
 `,
     )
-    .option('--manager <number>', 'Vault Manager', Number, 0)
-    .action(async opts => {
-      const { agoricNames, readLatestHead } = await tryMakeUtils();
-
-      const [metrics, quote] = await Promise.all([
-        readLatestHead(`published.vaultFactory.manager${opts.manager}.metrics`),
-        readLatestHead(`published.vaultFactory.manager${opts.manager}.quotes`),
-      ]);
-      const info = fmtMetrics(metrics, quote, values(agoricNames.vbankAsset));
-      show(info, true);
-    });
+    .option('--book <number>', 'Auction Book', Number, 0)
+    .action(
+      async (
+        /**
+         * @type {{
+         *   book: number,
+         * }}
+         */ opts,
+      ) => {
+        const { agoricNames, readLatestHead } = await tryMakeUtils();
+
+        /**
+         * ref publishSchedule()
+         * @see (import('@agoric/inter-protocol/src/auction/scheduler.js')} publishSchedule
+         *
+         * @typedef {import('@agoric/time/src/types.js').TimestampRecord} TimestampRecord
+         * @typedef {{
+         *   activeStartTime?: TimestampRecord
+         *   nextStartTime: TimestampRecord,
+         *   nextDescendingStepTime: TimestampRecord,
+         * }} ScheduleUpdate
+         *
+         * @typedef {{
+         *   startPrice: Ratio | null,
+         *   startProceedsGoal: Amount<'nat'> | null,
+         *   proceedsRaised: Amount<'nat'> | null,
+         *   startCollateral: Amount<'nat'>,
+         *   collateralAvailable: Amount<'nat'> | null,
+         *   currentPriceLevel: Ratio | null,
+         * }} BookUpdate
+         */
+        /** @type { [ScheduleUpdate, BookUpdate, *] } */
```

Comment:
I see another push after this discussion but I don't see the types imported. 
```suggestion
         */
        /** @type { [ScheduleNotification, BookDataNotification, *] } */
```

Reply:
yeah, I'm still working on it. thanks for watching carefully

Reply:
take another look, please? I hope you don't mind that I squashed all the fixups

Reply:
small enough that it's easy to fully re-review